config/crd/bases/kataconfiguration.openshift.io_kataconfigs.yaml

```diff
@@ -15,229 +15,232 @@ spec:
     plural: kataconfigs
     singular: kataconfig
   scope: Cluster
-  subresources:
-    status: {}
-  validation:
-    openAPIV3Schema:
-      description: KataConfig is the Schema for the kataconfigs API
-      properties:
-        apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: KataConfigSpec defines the desired state of KataConfig
-          properties:
-            config:
-              description: KataInstallConfig is a placeholder struct
-              properties:
-                sourceImage:
-                  description: SourceImage is the name of the kata-deploy image
-                  type: string
-              required:
-              - sourceImage
-              type: object
-            kataConfigPoolSelector:
-              description: KataConfigPoolSelector is used to filer the worker nodes
-                if not specified, all worker nodes are selected
-              properties:
-                matchExpressions:
-                  description: matchExpressions is a list of label selector requirements.
-                    The requirements are ANDed.
-                  items:
-                    description: A label selector requirement is a selector that contains
-                      values, a key, and an operator that relates the key and values.
+  versions:
```

Comment:
This part is hard to review for me just by looking at the diff. What are the important changes here to prevent multiple CRs?

Reply:
Autogenerated :)